src/App.js

```diff
@@ -1,25 +1,28 @@
-import logo from './logo.svg';
-import './App.css';
+import {Home, Deposit, Withdraw, Balance} from '../src/Components/'
```

Comment:
Why are you importing pages from `components` ??

Reply:
done